app/javascript/article-form/articleForm.jsx

```diff
@@ -85,33 +104,21 @@ export default class ArticleForm extends Component {
       logoSvg,
       helpFor: null,
       helpPosition: null,
+      ...previousContentState,
     };
   }
 
   componentDidMount() {
-    const { version, updatedAt } = this.state;
-    const previousContent =
-      JSON.parse(
-        localStorage.getItem(`editor-${version}-${window.location.href}`),
-      ) || {};
-    const isLocalstorageNewer =
-      new Date(previousContent.updatedAt) > new Date(updatedAt);
-
-    if (previousContent && isLocalstorageNewer) {
-      this.setState({
-        title: previousContent.title || '',
-        tagList: previousContent.tagList || '',
-        mainImage: previousContent.mainImage || null,
-        bodyMarkdown: previousContent.bodyMarkdown || '',
-        edited: true,
-      });
-    }
-
     window.addEventListener('beforeunload', this.localStoreContent);
   }
 
+  componentWillUnmount() {
+    window.removeEventListener('beforeunload', this.localStoreContent);
```

Comment:
Added this as it's a good practice to remove event listeners on component unmount.